### PR TITLE
[extractor/stripchat] fix unbound formats variable

### PR DESCRIPTION
**Pull request by freezboltz**
**IMPORTANT**: PRs without the template will be CLOSED

### Description of your *pull request* and other information

</details>

ADD DESCRIPTION HERE
fixed unbound formats variable

Fixes #

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

### CODE REVIEW

**Review by bashonly — CHANGES_REQUESTED** [review]
Revert all the formatting changes (indentation, double quotes)

The 2 suggestions I've made are the only necessary changes

File: yt_dlp/extractor/stripchat.py [hunk 1]
```diff
@@ -40,13 +40,16 @@ def _real_extract(self, url):
         server = traverse_obj(data, ('viewCam', 'viewServers', 'flashphoner-hls'), expected_type=str)
         model_id = traverse_obj(data, ('viewCam', 'model', 'id'), expected_type=int)
 
-        for host in traverse_obj(data, (
+        formats = []
+        for host in traverse_obj(data or [], (
```
Comment (bashonly) [inline yt_dlp/extractor/stripchat.py:44]:
```suggestion
        for host in traverse_obj(data, (
```
unnecessary

**Review by bashonly — APPROVED** [review]

**Comment by pro-ject143** [comment]
Any update? Still not able to download.

**Comment by pro-ject143** [comment]
Disappointing. Downloading off Stripchat is as a dead as a dodo.

**Pull request merged**